### PR TITLE
ci: add more fstypes support (xfs, jfs, f2fs)

### PR DESCRIPTION
**Pull request by jozzsi**
This PR is just a container change, no tests are added or removed.

## Checklist
- [ ] I have tested it locally
- [ ] I have reviewed and updated any documentation if relevant
- [ ] I am providing new code and test(s) for it

Fixes #

### CODE REVIEW

File: test/container/Dockerfile-Ubuntu [hunk 1]
```diff
@@ -6,6 +6,7 @@ RUN apt-get update -y -qq && apt-get upgrade -y -qq && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends -o Dpkg::Use-Pty=0 \
     asciidoc \
     astyle \
+    bcachefs-tools \
```
Comment (bdrung) [inline test/container/Dockerfile-Ubuntu:9]: bcachefs-tools was removed from Debian unstable and the future is unclear.

Comment (jozzsi) [reply test/container/Dockerfile-Ubuntu:9]: @bdrung what do you suggest we do ? 

As you can see I did not add it to the Debian container and I certainly hope we can have at least one container with bcachefs tools (it does not have to be Ubuntu).

If the CI container no longer builds due some distro packaging changes, it is trivial to remove.

There seems to be interest for a dedicated bcachefs dracut module also - https://github.com/dracut-ng/dracut-ng/issues/145

Comment (Conan-Kudo) [reply test/container/Dockerfile-Ubuntu:9]: I agree here, as it is, bcachefs is experimental and churning too much for use to test. I would leave it out for now. 
I think we're going to want to test ext4; xfs; and btrfs, as those are the three commonly used filesystems.

Comment (bdrung) [reply test/container/Dockerfile-Ubuntu:9]: I am fine with adding bcachefs-tools to the Ubuntu container, since it can be easily dropped when needed.

Comment (jozzsi) [reply test/container/Dockerfile-Ubuntu]: For now I removed bcachefs in the interest of making progress first with other fs types, like xfs .

**Review by LaszloGombos — APPROVED** [review]
thanks

**Pull request merged**